File: 241001/두 개의 격자 비교하기/compare-two-grid.py
a,b = list(map(int,input().split()));

arr = [list(map(int,input().split())) for _ in range(a)];
brr = [list(map(int,input().split())) for _ in range(a)];
Answer = [[0 for _ in range(b)] for _ in range(a)];

for i in range(a):
    for e in range(b):
        if(arr[i][e] == brr[i][e]):
            continue
        else:
            Answer[i][e] = 1

for i in Answer:
    for e in i:
        print(e, end=" ");
    print();

File: 250624/두 개의 격자 비교하기/compare-two-grid.js
const fs = require('fs');
const inputs = fs.readFileSync(0).toString().trim().split('\n');

const [n,m] = inputs[0].split(' ').map(Number);

const arr = Array(n).fill(0).map(() => Array(m));
const brr = Array(n).fill(0).map(() => Array(m));

for(let i = 1 ; i < inputs.length ; i++){
    inputs[i] = inputs[i].split(' ').map(Number);
}

for(let i = 1 ; i < inputs.length; i++){
    if(i < n+1){
        arr[i-1] = inputs[i];
    }else{
        brr[i-1-n] = inputs[i];
    }
}

for(let i = 0 ; i < n ; i++){
    let str = '';
    for(let e = 0 ; e < m ; e++){
        if(arr[i][e] == brr[i][e]){
            str += '0 ';
        }else{
            str += '1 ';
        }
    }
    console.log(str);
}